datafusion/common/src/parsers.rs

```diff
@@ -76,337 +72,3 @@ impl CompressionTypeVariant {
         !matches!(self, &Self::UNCOMPRESSED)
     }
 }
-
```

Comment:
The point of this PR is to remove this code (which has been upstreamed to arrow-rs)